### ISSUE TITLE
Unable to deserialize lockfile

### ISSUE THREAD

**Issue by pswaminathan** [theoremlp/multitool#27]
```
$ RUST_BACKTRACE=1 multitool --lockfile multitool.lock.json update
thread 'main' panicked at src/main.rs:206:41:
Unable to deserialize lockfile: Error("invalid type: string \"https://raw.githubusercontent.com/theoremlp/rules_multitool/main/lockfile.schema.json\", expected struct Binary", line: 2, column: 100)
stack backtrace:
   0: _rust_begin_unwind
   1: core::panicking::panic_fmt
   2: core::result::unwrap_failed
   3: multitool::main
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```

```
{
  "$schema": "https://raw.githubusercontent.com/theoremlp/rules_multitool/main/lockfile.schema.json",
...
```

I can provide the rest of the lockfile if needed, but I am not sure think it matters—even when I strip away everything else, it still fails. Looks like the parsing expects there to be only Binary declarations, not the `"$schema"`.

**Comment by mark-thm** [comment]
Released in 0.3.0.

**Comment by pswaminathan** [comment]
Thank you!
